runner/actions-runner.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+source logger.bash
+source graceful-stop.bash
+
```

Comment:
do we want to move `trap graceful_stop TERM` in here to make it more readable since it looks like `actopns-runner.sh` is the new main entrypoint script?
So, we can add `trap - TERM` at the end of the file to remove the trap?
You can ignore me, not a big deal. 😄

Reply:
Sounds much cleaner! Let me try. Thanks for your feedback 😄

Reply:
Addressed in a666a2f